services/masterdata/src/lib/components/masterdata-type-edit-form/masterdata-type-edit-form.component.html

```diff
@@ -1,4 +1,11 @@
-<lens-panel header="{{ formTitle }} Masterdata-type '{{ item?.name }}'">
+<lens-panel
+	header="{{
```

Comment:
suggestion: move this kind of logic to typescript to keep the html clean

Reply:
Good point! I've updated the code.